db/models/user.js

```diff
@@ -2,27 +2,48 @@
 
 // bcrypt docs: https://www.npmjs.com/package/bcrypt
 const bcrypt = require('bcryptjs')
-    , {STRING, VIRTUAL} = require('sequelize')
+    , {BOOLEAN, STRING, VIRTUAL} = require('sequelize')
 
 module.exports = db => db.define('users', {
-  name: STRING,
+  userName: STRING,
   email: {
     type: STRING,
     validate: {
       isEmail: true,
-      notEmpty: true,
     }
   },
+  firstName: STRING,
+  lastName: STRING,
+  shippingAddress: STRING ,
```

Comment:
Also -- style comment -- will want to remove the space after STRING on this line :)